src/data/cifar10.py

```diff
@@ -0,0 +1,46 @@
+import torch
+import torchvision
+import torchvision.transforms as transforms
+from torch.utils.data import Dataset
+from overrides import override
+
+from .dataset_interface import TaskDataset
+
+class CIFAR10(Dataset):
+    """ Modifies Torchvision's CIFAR10 dataset where:
+            images: downsampled images
+            labels: original images
+    """
+    def __init__(self, root, train=True, download=True, transform=None):
+        self.cifar10 = torchvision.datasets.CIFAR10(root=root, train=train, download=download, transform=None)
+        self.image_transform = transform
+        self.label_transform = transforms.Compose([transforms.ToTensor(), 
+                                                   transforms.Normalize((0.5,0.5,0.5),(0.5,0.5,0.5))
```

Comment:
Can you clarify how you chose (0.5, 0.5, 0.5)? I was checking the documentation and it isn't immediately clear to me